code/game/gamemodes/miniantags/guardian/types/protector.dm

```diff
@@ -8,15 +8,41 @@
 	tech_fluff_string = "Boot sequence complete. Protector modules loaded. Holoparasite swarm online."
 	bio_fluff_string = "Your scarab swarm finishes mutating and stirs to life, ready to defend you."
 	var/toggle = FALSE
+	/// The shields the guardian has, and brings with it as it moves
+	var/list/connected_shields = list()
 
 /mob/living/simple_animal/hostile/guardian/protector/ex_act(severity)
 	if(severity == 1)
-		adjustBruteLoss(400) //if in protector mode, will do 20 damage and not actually necessarily kill the summoner
+		adjustBruteLoss(200) //if in protector mode, will do 20 damage and not actually necessarily kill the summoner
 	else
 		..()
 	if(toggle)
 		visible_message("<span class='danger'>The explosion glances off [src]'s energy shielding!</span>")
 
+
+/mob/living/simple_animal/hostile/guardian/protector/Manifest()
+	. = ..()
+	if(toggle && cooldown > world.time)
+		var/dir_left = turn(dir, -90)
+		var/dir_right = turn(dir, 90)
+		connected_shields += new /obj/effect/guardianshield(get_step(src, dir_left), src, FALSE)
+		connected_shields += new /obj/effect/guardianshield(get_step(src, dir_right), src, TRUE)
+
+/mob/living/simple_animal/hostile/guardian/protector/Recall(forced)
+	. = ..()
+	if(cooldown > world.time && !forced)
+		QDEL_LIST_CONTENTS(connected_shields)
+
+/mob/living/simple_animal/hostile/guardian/protector/Move()
+	. = ..()
+	for(var/obj/effect/guardianshield/G in connected_shields)
+		var/dir_chosen
+		if(G.left_if_false_right_otherwise)
+			dir_chosen = turn(dir, 90)
+		else
+			dir_chosen = turn(dir, -90)
+		G.forceMove(get_step(src, dir_chosen))
```

Comment:
This can be simplified with the build-in `direct` var and removes the need for the `left_if_false_right_otherwise` var.
````dm
/mob/living/simple_animal/hostile/guardian/protector/Move(atom/newloc, direct, movetime)
	. = ..()
	for(var/obj/effect/guardianshield/G as anything in connected_shields)
		G.forceMove(get_step(fake_box, direct))

Reply:
Wait this var is for something else... I don't even know what's going on here.

Reply:
Hmm. Will try to improve the variable name for now